dpgen/remote/RemoteJob.py

```diff
@@ -731,22 +731,25 @@ def submit(self,
             status = self.check_status()
             if status in [  JobStatus.unsubmitted, JobStatus.unknown, JobStatus.terminated ]:
                 dlog.debug('task restart point !!!')
+                while self.check_limit(task_max=resources['task_max']):
```

Comment:
Can you add `if 'task_max' in resources and resources['task_max']>0:` in front of this line to make the default parameter is unlimited?

Reply:
OK, thanks.